src/sentry/monitors/tasks.py

```diff
@@ -101,10 +104,13 @@ def check_monitors(current_datetime=None):
         except Exception:
             logger.exception("Exception in check_monitors - mark missed")
 
+
+@instrumented_task(name="sentry.monitors.tasks.check_timeout", time_limit=15, soft_time_limit=10)
+def check_timeout(current_datetime=None):
     qs = MonitorCheckIn.objects.filter(
         status=CheckInStatus.IN_PROGRESS, timeout_at__lte=current_datetime
     ).select_related("monitor", "monitor_environment")[:CHECKINS_LIMIT]
-    metrics.gauge("sentry.monitors.tasks.check_monitors.timeout_count", qs.count())
+    metrics.gauge("sentry.monitors.tasks.check_timeout.count", qs.count())
```

Comment:
We'll have to make sure we update DD

Reply:
yeah that's a follow up task once this goes out